integration_tests/run_pyspark_from_build.sh

```diff
@@ -80,7 +80,11 @@ else
         # Make sure we have Parquet version >= 1.12 in the dependency
         LOWEST_PARQUET_JAR=$(echo -e "$MIN_PARQUET_JAR\n$PARQUET_HADOOP_TESTS" | sort -V | head -1)
         export INCLUDE_PARQUET_HADOOP_TEST_JAR=$([[ "$LOWEST_PARQUET_JAR" == "$MIN_PARQUET_JAR" ]] && echo true || echo false)
-        PLUGIN_JARS=$(echo "$TARGET_DIR"/../../dist/target/rapids-4-spark_*.jar)
+        if [ -e "$PLUGIN_JAR_PATH" ]; then
```

Comment:
why not just use PLUGIN_JARS and check if its set first? Though we can probably rename to PLUGIN_JAR at this point.

Also it would be nice to document, in the very least at the top of the file (which i realize is lacking any docs but good to make it better)

Reply:
Fixed. PTAL